includes/modules/searchandreplace/class-pb-search.php

```diff
@@ -37,7 +49,18 @@ function search_for_pattern( $search, $limit, $offset, $orderby ) {
 			if ( ! ini_get( 'safe_mode' ) ) {
 				set_time_limit( 0 );
 			}
-			return $this->find( '@' . preg_quote( $search, '@' ) . '@', $limit, $offset, $orderby );
+			if ( $this->regex ) {
+				// First test that the search and replace strings are valid regex
+				set_error_handler( array( &$this, 'regex_error' ) );
+				$valid = @preg_match( $search, '', $matches );
```

Comment:
`composer standards` doesn't show me any violations. Should I still add the line preemptively? Or is there something wrong with my setup and suppressing errors should trigger a validation error?